lua/nvim-treesitter/parsers.lua

```diff
@@ -229,6 +229,14 @@ list.erlang = {
   maintainers = { '@ostera' },
 }
 
+list.elixir = {
+  install_info = {
+    url = "https://github.com/ananthakumaran/tree-sitter-elixir",
+    files =  { "src/parser.c", "src/scanner.cc" },
+  },
+  maintainers = { '@nifoc' },
+}
```

Comment:
You have to add a requires _generate_from_source. If it's not in the repo it needs to be generated. You also have to check whether npm is required (e.g. for depdencies or whether tree-sitter-cli is sufficient.